### ISSUE TITLE
Chart for Richlist stats?

### ISSUE THREAD

**Issue by lathanbritz** [WietseWind/XRP-Stats-Frontend#4]
How can we chart the Richlist stats list. A line chart for each of those buckets would be grand. Any ideas where to start on this?

**Comment by WietseWind** [comment]
Definitely. You can use this tool to fetch and calculate the same info the tool fetches and calculates. https://github.com/WietseWind/xrp-ledgerstats

Use it for several ledger intervals, and plot it :)
